src/phpcs/Production/Sniffs/Abstract/MethodSniff.php

```diff
@@ -51,10 +51,10 @@ protected function hasMethodDocBlock(PHP_CodeSniffer_File $sniffedFile, $indexOf
     }
 
     /**
-     * Prüft, ob das Token an $index ein Whitespace ist.
+     * Checks if the token is a whitespace.
```

Comment:
Checks if the token at position of $index corresponds to a whitespace character.